packages/@uppy/svelte/package.json

```diff
@@ -25,11 +28,13 @@
     "dist"
   ],
   "exports": {
-    "./dist/styles.css": "./dist/styles.css",
-    ".": {
-      "types": "./dist/index.d.ts",
-      "svelte": "./dist/index.js"
-    }
+    ".": "./dist/index.js",
```

Comment:
You should read the docs, I don't think this is works now

https://svelte.dev/docs/kit/packaging#Anatomy-of-a-package.json-exports